File: test_api/test_edge_tts.js
import { Client } from "@gradio/client";

const client = await Client.connect("http://127.0.0.1:7861/");

const result = await client.predict(
  "/predict", {
    text: "Salut, ce mai faci?",
    voice: "ro-RO-AlinaNeural (Female)",
    rate: 0,
    volume: 0,
    pitch: 0,
});

console.log(result.data);

File: test_api/test_edge_tts.py
from gradio_client import Client

client = Client("http://localhost:7861")

result = client.predict(
    text="Salut, ce mai faci?",
    voice="ro-RO-AlinaNeural (Female)",
    rate=0,
    volume=0,
    pitch=0,
    api_name="/predict"
)

print(result)
